pr-data.csv

```diff
@@ -1083,13 +1083,13 @@ https://github.com/jenkinsci/remoting,abf0455a68ad6c52a57e912bb89d51f883f77542,.
 https://github.com/jenkinsci/remoting,abf0455a68ad6c52a57e912bb89d51f883f77542,.,hudson.remoting.DefaultClassFilterTest.testDefaultsOverrideExists,OD,,,https://github.com/TestingResearchIllinois/flaky-test-dataset/issues/82
 https://github.com/jenkinsci/remoting,abf0455a68ad6c52a57e912bb89d51f883f77542,.,hudson.remoting.DiagnosedStreamCorruptionExceptionTest.blockingStreamShouldNotPreventDiagnosis,OD,,,https://github.com/TestingResearchIllinois/flaky-test-dataset/issues/82
 https://github.com/jenkinsci/remoting,abf0455a68ad6c52a57e912bb89d51f883f77542,.,hudson.remoting.DiagnosedStreamCorruptionExceptionTest.exercise,OD,,,https://github.com/TestingResearchIllinois/flaky-test-dataset/issues/82
-https://github.com/jenkinsci/remoting,abf0455a68ad6c52a57e912bb89d51f883f77542,.,hudson.remoting.PrefetchingTest.testGetResource,ID,,,
-https://github.com/jenkinsci/remoting,abf0455a68ad6c52a57e912bb89d51f883f77542,.,hudson.remoting.PrefetchingTest.testGetResourceAsStream,ID,,,
-https://github.com/jenkinsci/remoting,abf0455a68ad6c52a57e912bb89d51f883f77542,.,hudson.remoting.PrefetchingTest.testGetResources,ID,,,
-https://github.com/jenkinsci/remoting,abf0455a68ad6c52a57e912bb89d51f883f77542,.,hudson.remoting.PrefetchingTest.testGetResources_precache,ID,,,
-https://github.com/jenkinsci/remoting,abf0455a68ad6c52a57e912bb89d51f883f77542,.,hudson.remoting.PrefetchingTest.testGetResource_precache,ID,,,
-https://github.com/jenkinsci/remoting,abf0455a68ad6c52a57e912bb89d51f883f77542,.,hudson.remoting.PrefetchingTest.testInnerClass,ID,,,
-https://github.com/jenkinsci/remoting,abf0455a68ad6c52a57e912bb89d51f883f77542,.,hudson.remoting.PrefetchingTest.testJarLoadingTest,ID,,,
+https://github.com/jenkinsci/remoting,abf0455a68ad6c52a57e912bb89d51f883f77542,.,hudson.remoting.PrefetchingTest.testGetResource,ID,DeveloperFixed,,
+https://github.com/jenkinsci/remoting,abf0455a68ad6c52a57e912bb89d51f883f77542,.,hudson.remoting.PrefetchingTest.testGetResourceAsStream,ID,DeveloperFixed,,
+https://github.com/jenkinsci/remoting,abf0455a68ad6c52a57e912bb89d51f883f77542,.,hudson.remoting.PrefetchingTest.testGetResources,ID,DeveloperFixed,,
+https://github.com/jenkinsci/remoting,abf0455a68ad6c52a57e912bb89d51f883f77542,.,hudson.remoting.PrefetchingTest.testGetResources_precache,ID,DeveloperFixed,,
+https://github.com/jenkinsci/remoting,abf0455a68ad6c52a57e912bb89d51f883f77542,.,hudson.remoting.PrefetchingTest.testGetResource_precache,ID,DeveloperFixed,,
+https://github.com/jenkinsci/remoting,abf0455a68ad6c52a57e912bb89d51f883f77542,.,hudson.remoting.PrefetchingTest.testInnerClass,ID,DeveloperFixed,,
+https://github.com/jenkinsci/remoting,abf0455a68ad6c52a57e912bb89d51f883f77542,.,hudson.remoting.PrefetchingTest.testJarLoadingTest,ID,DeveloperFixed,,
```

Comment:
For these tests, can you please add a note describing what SHA you ran Nondex on that found these tests to not be flaky anymore? 
https://github.com/TestingResearchIllinois/idoft#adding-notes